### DISCUSSION TITLE
Welcome to ZeroOctave Discussions!

### DISCUSSION THREAD

**Discussion by Astrodevil** [Zroctve/ZeroOctave.github.io#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jyoti-prakash-rout** [comment]
Hey, everyone!
I'm Jyoti, an Web Dev enthusiast pursuing my bachelor's in computer science and Engineering. I'm interested in Web Dev, Machine Learning, and DevOps and I always enjoy learning and exploring new things and stuff, as well as I'm a frontend developer with experienced building static web pages.
 
Looking forward to learning, collaborating, and connecting with all the amazing people in our community.

**Reply by Astrodevil** [reply]
Welcome to the community. 🙏

**Comment by himanshu1221** [comment]
Hi guys 👋
I am Himanshu chhatwal . I am a student of btech cse . I have a keen intrest in learning as well as practising Java , python , AI , JavaScript . my major interest currently is to learn WEB3 and explore the blockchain world as it fascinates me a lot and I am a person who is always eager to learn about the new technology and adapt them as quick as possible.I did an internship at immersephere as a web developer also contributed in open source . I look Forward to learn more and I am open to internships and projects related to my field....

**Comment by nithin999989** [comment]
Hi, everyone!
I am Nithin, an Web Dev enthusiast pursuing my bachelor's in computer science and Engineering. I'm interested in Web Dev, Machine Learning.I always enjoy learning and exploring new things and stuff.

**Comment by vishnu9865** [comment]
Hi, guys !
 - I am Jeyavishnu, a Full Stack Web Developer and Open Source Enthusiast currently pursuing 4th year of my BE, CSE degree. I am currently learning Typescript, Headless CMS, Frontend tech like postscript, less, scss, etc,..
 - I am excited to contribute and learn from this esteemed community.

**Comment by pranita28Dane** [comment]
- Creative and diligent aspiring Python Backend Developer with a specialization in Python backend. Currently, I recently graduated with a Masters of Computer Science postgraduate program from Pune University. I enjoy problem-solving and coding. Always strive to bring 100% to the work I do.
- Looking forward to learning, collaborating, and connecting with all the amazing people in our community.

**Comment by tothetop430** [comment]
Please add me to your organization

**Comment by bugbounted** [comment]
Please invite me to your organization

**Comment by thinkingdev9** [comment]
Please invite me to your organization

**Comment by OrionFable** [comment]
Please invite me to your organization

**Comment by NjorogeMaurice** [comment]
Please invite me to your community

**Comment by victorykop** [comment]
Please invite me to your organization

**Comment by Boris95326** [comment]
Please invite me to your organization

**Reply by Boris95326** [reply]
Hi.
Nice to meet you.
I'm a senior software engineer with 8 years of experience.
Looking forward to learning, collaborating, and connecting with all the amazing people in our community.

**Comment by RL-devone** [comment]
Please invite me to your community

**Comment by CodeHex083** [comment]
Please invite me to your community

**Comment by tomas0502** [comment]
I want to join this community.

**Comment by MMVonnSeek** [comment]
Hi my name is Max Muller, Please invite me to the GitHub Community Organization

**Comment by DarkHorse109707** [comment]
Hi.
Nice to meet you.
I'm a senior software engineer with 8 years of experience.
Looking forward to learning, collaborating, and connecting with all the amazing people in our community
